go.mod

```diff
@@ -14,13 +14,14 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/pflag v1.0.6
 	github.com/stretchr/testify v1.10.0
-	github.com/vishvananda/netlink v1.3.0
+	github.com/vishvananda/netlink v1.3.1-0.20250206174618-62fb240731fa
```

Comment:
Not sure this is needed but I think i ran `go mod tidy` instead of `go mod download` ?